tools/scripts/sync-version.js

```diff
@@ -0,0 +1,8 @@
+const pkg = require('../../package.json')
+const fs = require('fs')
+const path = require('path')
+
+const cdn_version = fs.readFileSync(path.join(__dirname, '../../', 'VERSION'), 'utf-8')
+const new_pkg_version = `0.${cdn_version.split('.').slice(0, 2).join('.')}`
```

Comment:
We may need to update this to account for the patch number from the `VERSION` file. Let's see what dempla and apm services say.

Reply:
DEM is 👍 on being able to parse this.  We should NOT publish pre-release versions to NRDB + connect for now.  prerelease versions should be for NPM only for now.